src/main/java/Duke.java

```diff
@@ -1,10 +1,75 @@
+import java.lang.reflect.Array;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws DukeException{
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        Scanner sc = new Scanner(System.in);
+        ArrayList<Task> list = new ArrayList<>();
+
+        while(sc.hasNext()) {
```

Comment:
Will it be better to abstract and break this method into many smaller units? It will help in the future also.